runtime/envfiles/envfilemonitor.go

```diff
@@ -17,11 +17,25 @@ import (
 )
 
 func MonitorEnvFiles(ctx context.Context) {
+	loadFile := func(file storage.FileInfo) error {
+		logger.Info(ctx).Str("filename", file.Name).Msg("Env file changed. Loading environment variables.")
+		err := LoadEnvFiles(ctx)
+		if err != nil {
+			logger.Err(ctx, err).Str("filename", file.Name).Msg("Failed to load env file.")
+			return err
+		}
+
+		return nil
+	}
 	sm := storage.NewStorageMonitor(".env", ".env.*")
+	sm.Added = loadFile
```

Comment:
This shouldn't have been needed.  `Changed` runs after any add, and we don't need to log the specific file that was changed.  So `Added` doesn't need to be set.